addons/website/static/src/js/tours/tour_utils.js

```diff
@@ -73,9 +73,12 @@ function changeImage(snippet, position = "bottom") {
 
 /**
     wTourUtils.changeOption('HeaderTemplate', '[data-name="header_alignment_opt"]', _t('alignment')),
+    By default, prevents the step from being active if a palette is opened.
+    Set allowPalette to true to selection options within a palette.
 */
-function changeOption(optionName, weName = '', optionTooltipLabel = '', position = "bottom") {
-    const option_block = `we-customizeblock-option[class='snippet-option-${optionName}']`
+function changeOption(optionName, weName = '', optionTooltipLabel = '', position = "bottom", allowPalette = false) {
+    const noPalette = allowPalette ? '' : '.o_we_customize_panel:not(:has(.o_we_so_color_palette.o_we_widget_opened))';
+    const option_block = `${noPalette} we-customizeblock-option[class='snippet-option-${optionName}']`;
```

Comment:
In fact this is not really stable for two reasons:
- You add a parameter to a function (even an optional one, better but not entirely stable)
- You change the default behavior for all changeOption call, meaning if someone used it to select something in a palette, it won't work anymore until they see your new parameter -> not ok.

Is this strictly necessary to fix in stable actually? Should not this be for master (real question)?

Reply:
Indeed you are right. I think it can be in master: producing the problem requires the user to stop following the tour exactly at one of the steps that could be hidden by a palette... and open the palette instead.
I will extract this specific commit to a dedicated PR on master - the other two will remain in this PR on 15.0.